packages/buckr/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "@textile/buckr",
+  "version": "1.0.0",
+  "description": "NodeJS utility for working with remote buckets.",
```

Comment:
pretty nice now with a few more commands. will use in npm scripts for a few more things (e.g. filecoin archives)